packages/devtools_app/lib/src/globals.dart

```diff
@@ -31,6 +31,13 @@ SurveyService get surveyService => globals[SurveyService];
 
 PreferencesController get preferences => globals[PreferencesController];
 
+String generateDevToolsTitle() {
+  if (!serviceManager.hasConnection) return '';
+  final connectedApp = serviceManager.connectedApp;
+  if (!connectedApp.appTypeKnown) return '';
+  return connectedApp.isFlutterAppNow ? 'Flutter DevTools' : 'Dart DevTools';
```

Comment:
`DevTools • Flutter` and `DevTools • Dart`?

`DevTools / Flutter` and `DevTools / Dart`?

`DevTools for Flutter` and `DevTools for Dart`?

Reply:
I'm in favor of `Flutter DevTools` and `Dart DevTools`. Since those are similar to `Chrome DevTools`, I think we can leverage some familiarity from our users.

Reply:
`DevTools • Flutter`  would be nice as we can display `DevTools` first and then only add the type of app being debugged once we know it. Flickering from `DevTools` to `Flutter DevTools` would be more distracting.
On the other hand I like `Flutter DevTools` and `Dart DevTools` better independent of the UI flickering constraints and as long as 95% of users never see the flicker it is probably fine.